### ISSUE TITLE
Set the dbname once

### ISSUE THREAD

**Issue by mikeymckay** [redgeoff/slouch#18]
It's a pain to have to specify the database name every function call. Is there a way to configure this during instantiation so that you don't have to type it every time?

**Comment by redgeoff** [comment]
Hmm... we've found the opposite: that you want the flexibility to set the DB name without having to create an instance that is bound to the DB name, like is done in nano. We've made this easier by creating a variable like

```js
var dbName = 'mydb';
```

and then using it in all the calls, e.g.

```js
slouch.doc.create(dbName, ...);
```

Let me ponder this a little to see if there is some middle ground that isn't as restrictive as nano's approach. Maybe a "current/default" DB name would work.

**Comment by redgeoff** [comment]
Yes @mariosant, a curried function would work well here! Would you like to take a stab at creating a PR that adds this? Something like
```
const myDB = slouch.use('dbName');
```
which would then allow for things like:
```
myDB.doc.create()
myDB.db.changes()
...
```
A API reference is available at https://github.com/redgeoff/slouch/wiki#table-of-contents
